src/main/java/org/cloudfoundry/promregator/scanner/ReactiveAppInstanceScanner.java

```diff
@@ -182,11 +183,14 @@ public void setNumberOfInstances(int numberOfInstances) {
 				// NB: This drops the current target!
 				return Mono.empty();
 			}
-			
-			v.setApplicationId(sas.getId());
-			
+
 			List<String> urls = sas.getUrls();
 			if (urls != null && !urls.isEmpty()) {
+				/*
```

Comment:
This comment looks misplaced to me.

Reply:
Since the ID assignment is removed. I'm not certain where to place this either. I could put it at the point where we retrieve the SAS. Or just remove it.

Reply:
I'd put it [after line 186](https://github.com/promregator/promregator/pull/140/files#diff-eac88078a44f3aaf65fdd4858abe390bR186), so right before the first usage of any attribute, but after it's clear that it has been initialized properly. 

What do you think?